app/src/main/java/at/roteskreuz/stopcorona/screens/debug/exposure_notifications/DebugExposureNotificationsViewModel.kt

```diff
@@ -174,14 +164,21 @@ class DebugExposureNotificationsViewModel(
         getTemporaryExposureKeyHistory()
     }
 
+    suspend fun asyncGetTemporaryExposureKeyHistory(): MutableList<TemporaryExposureKey>{
```

Comment:
Just info: async "keyword" is for `async { }: Deffered<T>`, which means, the caller should call `await()`. Your implementation is clear suspending getter.